src/components/UriViewer.js

```diff
@@ -1,11 +1,13 @@
+// noinspection RegExpUnnecessaryNonCapturingGroup
```

Comment:
If this is for an intellij inspection, I'd rather not commit it into the code. Is it possible for you to just disable the inspection for yourself?

Reply:
It is ultimately for any developer. It helps visibly linting. If the file is ✅ when you start, and not when you finish, you likely should fix your changes before committing.

We could also just fix the unused regex parts. BT is very ticket happy so I'll submit a Jira for follow up and then BT can refine, prioritize, etc.

Reply:
Would it be crazy to just run the intention to fix the regex in this PR? I'd rather that than disabling an inspection, if we're going to be changing lines for it anyway.